codegen-test/model/rest-json-extras.smithy

```diff
@@ -113,7 +114,8 @@ structure StringPayloadInput {
     uri: "/primitive-document",
     method: "POST",
     body: "{}",
-    params: {}
+    params: {},
+    appliesTo: "client",
```

Comment:
```suggestion
    headers: { "Content-Type": "application/json" },
```

This test fails for the server because it's an invalid restJson1 request. This should fix this test and the other ones where I've made the same suggestion.

Place `headers` below `body` though like in the awslabs/smithy tests; the GitHub UI does not allow me to make that suggestion.